lib/hammer_cli/ca_cert_manager.rb

```diff
@@ -0,0 +1,71 @@
+require 'fileutils'
+
+module HammerCLI
+  class CACertManager
+    attr_reader :ca_path
+
+    def initialize(ca_path)
+      @ca_path = ca_path
+    end
+
+    def add_ca_cert(uri)
+      raw_cert = get_ca_cert(uri)
+      cert_file = cert_file_name(uri)
+      store_ca_cert(raw_cert, cert_file)
+    end
+
+    def get_ca_cert(uri)
```

Comment:
My suggestion is to remove `get_ca_cert` and `noverify_ssl_context` out of `CertManager`. Yes, it would be more or less just wrapper around `SSLSocket` and that's imho the point - get rid of the dependency. `CertManager` will only handle installing certs (string) into given location and won't care where they come from.

I find injecting values into tested objects much cleaner than stubbing methods on the tested objects (you never know what you stub out).

But if you insist on having it in one class, I'm ok. It's not any huge monolith. My only concern was that it breaks good practice;) We can refactor it later if there's need for that.